File: Python3/crescenteAB.py
"""
Problema: Crie duas variáveis (A e B) e leia 1 valor para cada um deles.
Disponibilize eles de forma crescente."""

a = int(input("Digite o valor de A: "))
b = int(input("Digite o valor de B: "))
if (a > b):
   print("Em ordem crescente B = %d - A = %d" %(b,a))
else:
   print("Em ordem crescente A = %d - B = %d" %(a,b))
 

File: java/crescenteAB.java
import java.util.Scanner;

class crescenteAB {
   public static void main(String args[]) {
      Scanner sc = new Scanner(System.in);
      int a, b;
      System.out.println("digite o valor de A");
      a = sc.nextInt();
      System.out.println("digite o valor de B");
      b = sc.nextInt();
      if (a > b)
         System.out.println("Em ordem crescente B=" + b + " - A=" + a);
      else
         System.out.println("Em ordem crescente A=" + a + " - B=" + b);

      sc.close();
   }
}
